Generator/Sources/NeedleFramework/Generating/Serializers/DependencyProviderInitBodySerializer.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Copyright (c) 2018. Uber Technologies
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+/// A serializer that produces the initializer body code for the dependency
+/// provider.
+class DependencyProviderInitBodySerializer: Serializer {
+
+    /// Initializer.
+    ///
+    /// - parameter provider: The provider to generate initializer body
+    /// code for.
```

Comment:
nit: I'd say `source code` as you do below.

Reply:
Done.